smithy-aws-protocol-tests/model/restJson1/json-maps.smithy

```diff
@@ -286,6 +286,8 @@ structure JsonMapsInputOutput {
     sparseNumberMap: SparseNumberMap,
     sparseBooleanMap: SparseBooleanMap,
     sparseStringMap: SparseStringMap,
+    denseSetMap: DenseSetMap,
```

Comment:
Looks like you added the members to the structures, but didn't update the test contents to utilize them.